metal/loadbalancers/metallb/configmap_test.go

```diff
@@ -64,7 +64,9 @@ func TestConfigFileAddPeerByService(t *testing.T) {
 
 		cfg.Peers = peers[:]
 		peers := []Peer{tt.peer}
-		m.UpdatePeersByService(context.Background(), &peers, tt.addServiceNamespace, tt.addServicename)
+		if _, err := m.UpdatePeersByService(context.Background(), &peers, tt.addServiceNamespace, tt.addServicename); err != nil {
```

Comment:
Here we're making the method call _and_ checking for an error in the `if` condition, but in other places in this PR (e.g. [`metal/eip_controlplane_reconciliation.go`, line 155](https://github.com/equinix/cloud-provider-equinix-metal/pull/371/files#diff-e21bc796cf3fcaa8de618836c7670a9fe7bb512313c6328f63aa66f328e9d462R155) we're making the method call outside of the `if` condition.  Is this a style preference?  Or is it prescribed by the linter?

Reply:
Ok, I've swapped it over to the if _, err, style.